test-alerts.cpp

```diff
@@ -2,7 +2,71 @@
 
 #include "test/catch.hpp"
 #include "typewise-alert.h"
+#include "string.h"
 
-TEST_CASE("infers the breach according to limits") {
-  REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
+TEST_CASE("infers the breach according to limits")
```

Comment:
Good set of test cases 👍 Almost all combinations are covered!

Reply:
thank you